### ISSUE TITLE
Single black bar

### ISSUE THREAD

**Issue by massanchik** [prime31/Nez#592]
I'm on OS X, trying to setup simple scene.
Single black bar showing on the right side of the viewport with any game resolution.
![Снимок экрана 2020-05-31 в 16 03 04](https://user-images.githubusercontent.com/3767814/83353067-9629e900-a358-11ea-8e12-e6c10cc30e8d.png)

```c#
public class Game1 : Core
    {

        public Game1(): base(100, 100)
        {
        }

        protected override void Initialize()
        {
            base.Initialize();
            
            var scene = Scene.CreateWithDefaultRenderer(Color.CornflowerBlue);

            var txCrate = scene.Content.LoadTexture("crate");
            var entity = scene.CreateEntity("crate");
            entity.AddComponent(new SpriteRenderer(txCrate));
            entity.Transform.Position = new Vector2(50, 50);
            
            Core.Scene = scene;
        }
    }
```

**Comment by massanchik** [comment]
I've tested a blank monogame project and there is no black bar

**Comment by massanchik** [comment]
![Снимок экрана 2020-05-31 в 16 20 22](https://user-images.githubusercontent.com/3767814/83353363-b8bd0180-a35a-11ea-9404-e4574202aca0.png)
```c#
entity.Transform.Position = new Vector2(96, 50);
```

**Comment by massanchik** [comment]
Maybe problem with screen size calculation
Bar is gone when added +1 to the width Nez\ECS\Scene.cs : 577
```c#
var screenSize = new Point(Screen.Width + 1, Screen.Height);
```
![Снимок экрана 2020-05-31 в 16 41 43](https://user-images.githubusercontent.com/3767814/83353814-a6909280-a35d-11ea-975c-80093d0df5d6.png)

**Comment by massanchik** [comment]
Also there is division by zero without errors in Scene.cs
![Снимок экрана 2020-05-31 в 16 48 11](https://user-images.githubusercontent.com/3767814/83353951-b066c580-a35e-11ea-84b2-260b35522ec9.png)

**Comment by massanchik** [comment]
Same problem with android emulator
![Снимок экрана 2020-05-31 в 21 42 06](https://user-images.githubusercontent.com/3767814/83360148-bc1ab200-a387-11ea-869f-3cf92421d9a6.png)

**Comment by xdrie** [comment]
Are you using .NET Core (for macOS)? i'm assuming the android emulator is via Mono/Xamarin.Android.

if so, check if it's related to [this issue](https://github.com/prime31/Nez/issues/571#issuecomment-619463535), and here is [instructions for a fix](https://github.com/prime31/Nez/issues/571#issuecomment-619464856)

If you are using Mono and not .NET Core, however, this is probably a different bug.

**Comment by massanchik** [comment]
net48 for mac and Mono/Xamarin.Android 
problem exists on both devices

**Comment by massanchik** [comment]
So far I've tracked values to draw method call Nez\ECS\Scene.cs : 543
```c#
Graphics.Instance.Batcher.Draw(currentRenderTarget, _finalRenderDestinationRect, Color.White);
```
although _finalRenderDestinationRect values are correct, black bar appears anyway.
but if you increase _finalRenderDestinationRect.Width - bar is gone

**Comment by massanchik** [comment]
Enabled resize on mac, and on some window sizes bar is not present.
And it doesn't matter what was resized, only width or only height.
Maybe it's rounding problem with aspect ratio or scaling calculations

**Comment by massanchik** [comment]
But it's strange. Bar is present with 400 by 400 res, with ration of 1.

**Comment by massanchik** [comment]
This issue related to https://github.com/MonoGame/MonoGame/issues/7123
Projects was on 3.8.. version of monogame. 
On desktop it was fixed by downgrading to 3.7, however on android project refused to build on that version. 
So 3.8 for monogame.android with hacks from issue above
